kubernetes/lib/sar.rego

```diff
@@ -0,0 +1,36 @@
+package kubernetes.lib
+
+import data.kubernetes.lib
+
+sar_raw(resourceAttributes, user, groups) = output {
+	body := {
+		"apiVersion": "authorization.k8s.io/v1",
+		"kind": "SubjectAccessReview",
+		"spec": {
+			"resourceAttributes": resourceAttributes,
+			"user": user,
+			"groups": groups
+		}
+        }
+	req := {
+		"method": "post",
+		"url": "http://127.0.0.1:8001/apis/authorization.k8s.io/v1/subjectaccessreviews",
+		"headers": {
+			"Content-Type": "application/json"
+		},
+		"body": body
+	}
+	http.send(req, output)
+}
+
+sar_allowed(resourceAttributes, user, groups) = allowed {
```

Comment:
If you're using the `input` document as described above, this will turn into ...
```
sar_allowed = allowed {
    output := sar_raw
    .... as before
}